Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/README.md

```diff
@@ -31,10 +26,14 @@ The function for the single qubit gates requires that the user expresses the cal
 
 ```python
 def bake_phased_xz(baker: Baking, q, x, z, a):
-    element = f"q{q}_xy"
-    baker.frame_rotation_2pi(-a, element)
+    if q == 1:
+        element = f"q{q1_idx_str}_xy"
+    else:
+        element = f"q{q2_idx_str}_xy"
```

Comment:
Nope- it's taken globally. It's better this way for transparency, since any function calls to these bakery functions is made from the internal TwoQubitRB class